src/frontend/src/pages/Calendar/SummaryModal/SummaryComponents/DesignReviewDelayModalDetailsButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box } from '@mui/system';
+import { DesignReview } from 'shared';
+import { Link as RouterLink } from 'react-router-dom';
+import { routes } from '../../../../utils/routes';
+import NERFailButton from '../../../../components/NERFailButton';
+
+interface DesignReviewSummaryModalDetailsButtonProps {
+  designReview: DesignReview;
+}
+
+const DesignReviewSummaryModalDetailsButton: React.FC<DesignReviewSummaryModalDetailsButtonProps> = ({ designReview }) => {
+  return (
+    <Box
```

Comment:
you should be able to incorporate this code into `DesignReviewSummaryModalButtons` and move all the logic about which buttons to show into that component